BackendNew/scripts/create-tables.sql

```diff
@@ -30,4 +30,27 @@ CREATE TABLE Company (
     PRIMARY KEY (CompanyId),
     FOREIGN KEY (RoleId) REFERENCES Role(RoleId),
     FOREIGN KEY (CreatedBy) REFERENCES Admin(AdminId)
+);
+
+CREATE TABLE Batch (
+    BatchId int NOT NULL AUTO_INCREMENT,
+    CreatedBy int NOT NULL,
+    RoleId int NOT NULL,
+    Description varchar(255),
+    Yeah int,
+    StartDate DATE,
+    EndDate DATE,
+    CompanyCount int NOT NULL,
+    PRIMARY KEY (BatchId),
+    FOREIGN KEY (CreatedBy) REFERENCES Admin(AdminId)
```

Comment:
Do we need this foreign key? Because batches are always added by one super admin as of now.

Reply:
I added it because of this ER diagram or else we can remove it.

![imsdb](https://user-images.githubusercontent.com/32588172/81556881-afb6c100-93a8-11ea-8f16-459b2fbce6f1.png)

Reply:
can we remove it?

Reply:
yes I will remove it